II.pass-tests/05-factorial/factorial.js

```diff
@@ -8,8 +8,13 @@
 // calculate and return the factorial of int
 // note: factorial of 0 is 1
 
-function factorial(int) {
-
+function factorial (int) {
+  if (int >= 0) {
+    var fact = 1
+    for (i = 2; i <= int; i++) {
+      fact = fact * i
```

Comment:
You could make this more concise with fact =* i (this just translates to 'mutiply myself with i')